### PR TITLE
split type, scheme and company

### LINKED ISSUE #25: Split type & company name for searching
~We should set the `alt` attribute for each logo to be the same as the title, and instead use a `data-type` attribute to do the filtering.~

EDIT: updating this issue as I realise that the alt is actually what it should be! It just contains the `type`. I think we should split the type and brand name up, however? And have a different attribute to search for?

Currently, if you filter for `white` and type "white" (or any of its characters) in the search bar, you will just get a grid of all the logos, as they all contain that string. Also if a brand has "white" in it's name, all of it's colours will show up when filtering for "white"

To solve this we could have another datapoint in the cms that's just the company name? we would use this for searching, the type for filtering, and both for the image alt:

```
<img alt={title_with_type} data-type={type} data-brand={brand} />
```

### PR DESCRIPTION
**Pull request by neefrehman**
- Splits brand name, type, and colourScheme into three data points to add as `data-` attributes to the images
- uses those data attributes for searching and filtering

NOTE: I've renamed `type` to `colourScheme`, and have used `logoType` (logo, icon etc) as `type` in the cms upload script and in the frontend. i think this avoids confusion in anticipation of uploading the new icons?

NOTE 2: I haven't run the upload script yet. @kemiljk I think we might need to purge the current cms before doing it due to the conflicting `type` property name? And also because these logos are already uploaded so it would skip them. If you're happy with these changes and clearing the current CMS, feel free to run the script. Probably some debugging to do so let me know if there are any errors :)

closes #25

### CODE REVIEW

File: scripts/upload-assets.ts [hunk 1]
```diff
@@ -118,47 +127,60 @@ const uploadLogo = async ({
...
-                title: "Svg",
+                title: "Brand name",
+                key: "brand_name",
                 type: "text",
-                key: "svg",
-                value: fs.readFileSync(path, "utf-8")
+                value: brandName,
             },
             {
+                title: "Colour Scheme",
```
Comment (kemiljk) [inline scripts/upload-assets.ts:153]: Suggest this is "Colour scheme" for consistency with "Brand name" @neefrehman.

**Comment by kemiljk** [comment]
Awesome, so I've looked it over and suggested one change. I'm also running the build script currently to see how it gets on. So far, so good. One thing I did notice, if the code detects there are no matches (i.e. no published items in the CMS at all) it just fails with a 404. I don't know if we can do something to say that if `getLogosFromCms === null => skip` kind of thing?

**Comment by neefrehman** [comment]
I've just added that check for length. And what change did you suggest? I can't see it!

**Comment by kemiljk** [comment]
Oops sorry, I forgot I had to hit Suggest! @neefrehman

**Comment by neefrehman** [comment]
got it @kemiljk! I think you'll have to purge the cms and rerun the script again now, though

**Comment by neefrehman** [comment]
@kemiljk did the upload work ok? I'm getting some errors in the plugin right now. looks like the metadata fro brandName and colour aren't in the cms? Or haven't been fetched properly

**Comment by neefrehman** [comment]
@kemiljk crisis over! I was using camelCase instead of snake_case for the metadata properties. just pushed a fix, looks like its working well :)

**Pull request merged**